optimade/filtertransformers/mongo.py

```diff
@@ -307,11 +368,7 @@ def replace_with_relationship(subdict, prop, expr):
                     subdict["$and"] = []
                 subdict["$and"].extend(
                     [
-                        {
-                            f"relationships.{_prop}.data": {
-                                "$size": expr.pop("$size"),
-                            }
-                        },
+                        {f"relationships.{_prop}.data": {"$size": expr.pop("$size")}},
```

Comment:
Same here with whitespace

Reply:
In this case, the first line gets broken up while the second line is not. Which seems a bit arbitrary to me. The first line is a bit longer than the second, but still easily fits on one line. I have reverted it to the multi line version, though.

Reply:
The point isn't really what the whitespace actually is, just that it has changed and will be lumped in with the other changes in this PR. This will just be an artifact of `black` automatically splitting long lines, but not recombining them if they become shorter again

Reply:
I tried to undo these whitespace changes, but black automatically does them again.

Reply:
Hmm, that's strange. Are you using it via pre-commit and the pinned version number?

Reply:
With the pre-commit Black, the code is not changed. 
It seems like it used a different version of Black than the version that is executed from the command line. I have updated Black to version 21.6b0, and now it no longer seems to change the white space.

Reply:
The pre-commit black did not change the code. I have now updated the local version of black to 21.6b0, and it seems the white space is no longer changed.

Reply:
Awesome, thanks for doing that (and sorry for being picky)